docs/tutorial/tutorial-3.rst

```diff
@@ -438,7 +439,27 @@ or doing other pre-distribution tasks.
 
       [helloworld] Packaged dist/helloworld-0.0.1-1.fc36.x86_64.rpm
 
-    The ``dist`` folder will contain the RPM file that was generated.
+    The ``dist`` folder will contain the ``.rpm`` file that was generated.
+
+    If you're on an Arch-based distribution, you'll see:
+
+    .. code-block:: console
+      
+      (beeware-venv) $ briefcase package
+
+      [helloworld] Finalizing application configuration...
+      Targeting manjaro:rolling (Vendor base arch)
```

Comment:
Manjaro is a bit of a niche fork of Arch; better to keep this vanilla.
```suggestion
      Targeting arch:rolling (Vendor base arch)
```

Reply:
Done.